index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Global Summit</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="mobile-navbar">
```

Comment:
- [ ] As shown in the images below on mobile view you are to only show the hamburger menu icon, but from your design your logo is also displaying. please kindly follow the provided project design 👍 
- project design
![cc](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/2023172a-f45f-4b8e-a6b3-dc2aca201b42)
- your design
![cc ham](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/c5dd1bf6-26b0-4cdb-95c8-c9512ae89ee5)